splinter/driver/webdriver/phantomjs.py

```diff
@@ -35,7 +35,13 @@ def __init__(self, user_agent=None, load_images=True,
         if desired_capabilities:
             capabilities.update(desired_capabilities)
 
-        self.driver = PhantomJS(desired_capabilities=capabilities, **kwargs)
+        if executable_path is not None:
```

Comment:
instead to add `if` and `else` you cand add `executable_path` to kwargs parameters, something like this:

```
if executable_path:
    kwargs["executable_path"] = executable_path
```

btw this already works without this code, because splinter already gets custom parameters with **kwargs and pass it to PhantomJS.